docs/source/getting_started/index.md

```diff
@@ -183,49 +150,142 @@ response = client.inference.chat_completion(
 print(response.completion_message.content)
 ```
 
-To run the above example, put the code in a file called `inference.py`, ensure your `conda` or `virtualenv` environment is active, and run the following:
 ```bash
-pip install llama_stack
-llama stack build --template ollama --image-type <conda|venv>
-python inference.py
+python lstest.py
```

Comment:
added file names as comments so that they can be run with that name - maybe not needed?

Reply:
Actually, not a bad idea! Users that try this and find issues will probably refer to the script this way.